classes/Dealer.js

```diff
@@ -0,0 +1,42 @@
+import { StandardDeck } from './Deck'
+
+class Dealer {
+  constructor(deck) {
+    this.deck = deck
+  }
+
+  shuffle() {
+    const plusMinus = (Math.random() > 0.5) ? 1 : -1
+    const cut = Math.floor(Math.random() * 5) * plusMinus
+    const leftHand = this.deck.cards.splice(0, (this.deck.cards.length / 2) + cut)
+    const rightHand = this.deck.cards.splice(0, this.deck.cards.length)
+    
+    while (leftHand.length && rightHand.length) {
+      let mult = 0.5
+      let hand = Math.random() > mult ? 'left' : 'right'
+      if (hand === 'left') {
+        this.deck.cards.push(leftHand.shift())
+        mult += 0.1
+      } else {
+        this.deck.cards.push(rightHand.shift())
+        mult -= 0.1
+      }
+    }
+
+    while (leftHand.length) {
+      this.deck.cards.push(leftHand.shift())
+    }
+
+    while (rightHand.length) {
+      this.deck.cards.push(rightHand.shift())
+    }
+  }
+}
```

Comment:
Wait, I don't mean map here, cause that would just return a duplicate array. You could use a forEach here, or just concat the 2 arrays.

Reply:
1. I agree with the idea, but I think it should be a `forEach` rather than a `map` since I'm not actually mapping anything. 

2. I don't really agree with this one. I'm actually not sure what you are suggesting as the alternative.
    Assuming you mean that I should just concat them, I don't want to do this, because I'm pretty sure that would mess with the order. 
    If you mean that I just create a new card with the same values, it is definitely less realistic and I would assume it is actually less efficient that what I'm doing.
Either way, Array.prototype.shift() is a constant time operation, and at most I would be doing maybe 4 or 5 of these per shuffle, so I think it is fine

Reply:
1. Yeah I already noticed it shouldn't be map and commented again before this. 
2. Also mentioned concat in the previous comment, but anyway concat returns in order. You don't need to create a new card. You can do `this.deck.cards = this.deck.cards.concat(_hand_)`